scss/forms/_form-check.scss

```diff
@@ -246,144 +246,15 @@
   clip: rect(0, 0, 0, 0);
   pointer-events: none;
 
-  // Boosted mod: custom styles for .btn-check
-  // @note almost the same structure than the button-variant() mixin
-  // @note meant to override whatever .btn-* variant is used
-  + .btn {
-    margin-left: 0 !important; // stylelint-disable-line declaration-no-important
-    color: $black;
-    @include gradient-bg($white);
-    border-color: $gray-500;
-    @include box-shadow($btn-box-shadow);
-
-    &.btn-icon:not(.btn-no-outline)::before,
-    &.btn-icon:not(.btn-no-outline)::after {
-      display: inline-block;
-      width: var(--#{$prefix}border-width);
-      height: 100%;
-      content: "";
-    }
-
-    .btn-group &:not(.btn-no-outline):not(:first-of-type) {
-      border-left: 0;
-    }
-
-    .btn-group &:not(.btn-no-outline):not(:last-of-type) {
-      border-right: 0;
-    }
-  }
-
-  &:hover + .btn,
-  + .btn:hover {
-    color: $accessible-orange;
-    background-color: $white;
-    border-color: $gray-500;
-  }
-
-  .btn-group & + .btn:hover {
-    z-index: 0;
-  }
+  // Boosted mod: no disabled state
 
+  // Boosted mod
   &[data-focus-visible-added]:focus {
     + .btn,
     + .tag {
       @include focus-visible();
     }
   }
-
-  &:checked {
-    + .btn {
-      color: $white;
-      background-color: $black;
-      // Remove CSS gradients if they're enabled
-      background-image: if($enable-gradients, none, null);
-      border-color: $black;
-    }
-
-    + .btn-no-outline {
-      &,
-      &:hover {
-        color: $accessible-orange;
-        background-color: transparent;
-      }
-    }
-
-    + .tag {
-      color: var(--bs-tag-active-color);
-      background-color: var(--bs-tag-active-decoration-color);
-      border-color: var(--bs-tag-active-decoration-color);
-    }
-  }
-
-  &:active + .btn,
-  + .btn:active,
-  + .btn.active {
-    color: color-contrast($primary);
-    background-color: $primary;
-    // Remove CSS gradients if they're enabled
-    background-image: if($enable-gradients, none, null);
-    border-color: $primary;
-  }
-  // End mod
-
-  &[disabled],
```

Comment:
It's not possible to keep this rule as Bootstrap so that we can keep also `$form-check-btn-check-disabled-opacity`?